test/report.test.js

```diff
@@ -131,3 +157,9 @@ function dependencyTreeScarfEnabled () {
 
   return parsedLsOutput
 }
+
+function dependencyTreeTopLevelInstall () {
+  const exampleLsOutput = fs.readFileSync('./test/top-level-package-ls-output.json')
+  const parsedLsOutput = JSON.parse(exampleLsOutput)
+  return parsedLsOutput
+}
```

Comment:
```
tim@sirius ~/s/scarf-js> npm test

> @scarf/scarf@1.0.6 test /home/tim/src/scarf-js
> jest

 PASS  test/report.test.js
  Reporting tests
    ✓ Logging rate limit check (53ms)
    ✓ Redact sensitive data (3ms)
    ✓ Intermediate packages can disable Scarf for their dependents (2ms)
    ✓ Disable when package manager is yarn (1ms)
    ✓ Configurable for top level installs, and enabled for global installs (2ms)

  console.log report.js:49
    Error: ENOENT: no such file or directory, open './scarf-js-test-history.log'
        at Object.openSync (fs.js:458:3)
        at Object.readFileSync (fs.js:360:35)
        at Object.getRateLimitedLogHistory (/home/tim/src/scarf-js/report.js:451:29)
        at Object.<anonymous> (/home/tim/src/scarf-js/test/report.test.js:30:26)
        at Object.asyncJestTest (/home/tim/src/scarf-js/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:100:37)
        at /home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:45:12
        at new Promise (<anonymous>)
        at mapper (/home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:28:19)
        at /home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:75:41 {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: './scarf-js-test-history.log'
    }

  console.log report.js:49
    Error: ENOENT: no such file or directory, open './scarf-js-test-history.log'
        at Object.openSync (fs.js:458:3)
        at Object.readFileSync (fs.js:360:35)
        at getRateLimitedLogHistory (/home/tim/src/scarf-js/report.js:451:29)
        at Object.rateLimitedUserLog (/home/tim/src/scarf-js/report.js:439:19)
        at Object.<anonymous> (/home/tim/src/scarf-js/test/report.test.js:32:12)
        at Object.asyncJestTest (/home/tim/src/scarf-js/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:100:37)
        at /home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:45:12
        at new Promise (<anonymous>)
        at mapper (/home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:28:19)
        at /home/tim/src/scarf-js/node_modules/jest-jasmine2/build/queueRunner.js:75:41 {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: './scarf-js-test-history.log'
    }

  console.log report.js:442
    A user message

Test Suites: 1 passed, 1 total
Tests:       5 passed, 5 total
Snapshots:   0 total
Time:        0.788s, estimated 1s
Ran all test suites.
tim@sirius ~/s/scarf-js> 
```

Reply:
Yeah, this is expected. That history file is to ensure we don't spam with messages, but in tests that file isn't there yet. We log that trace (only in verbose mode) for observability but it is the expected behavior. 

Do you think we should change this?

Reply:
Sorry that I'm late in responding. Please bug me if I miss a comment. I am not at all bothered by this. I just noticed it. The tests still report passing so :LGTM: